loadero_python/resources/file.py

```diff
@@ -40,6 +42,22 @@ def __init__(
         content: str or None = None,
         password: str or None = None,
     ):
+        """Creates a new FileParams instance that will contain single file
+        resources attributes.
+
+        Args:
+            file_id (int, optional): Existing file resources ID.
+            Defaults to None.
+
+            file_type (FileType, optional): The file resources type.
+            Defaults to None.
+
+            content (str, optional): Content of file resource. Defaults to None.
+
+            password (str, optional): Set or provide a password for file.
```

Comment:
Can be little bit misleading. By providing the password you do not "set" the password for it as in securing it, you can only provide the password for it to be used if the usage requires it.